### PR TITLE
Migration itemContextMenu and maintabmanager to ES6 modules

### PR DESCRIPTION
**Pull request by Camc314**
**Changes**
converted itemContextMenu and maintabmanager to ES6 modules.

Part of jellyfin#868

### CODE REVIEW

**Comment by Influence365** [comment]
> I recently rewrote the skin manager from scratch in another open pull request, everything else looks fine though.

Oh right, I didn't see that, do you want me to remove the changes for skin manager?

**Comment by dkanada** [comment]
I also migrated the file to ES6 during the rewrite so removing them here should be fine, sorry for the trouble.

**Comment by Influence365** [comment]
No worries, I'll remove it, thanks for letting me know

**Comment by dkanada** [comment]
@Influence365 I forgot there would be merge conflicts if we targeted this branch. Could you rebase your commits against the ES6 branch? We could review and merge them now with that target since master is currently frozen.

**Comment by Influence365** [comment]
ahh, why does it do this when i rebase with es6 as head

**Comment by dkanada** [comment]
Hmmm maybe cherry picking the specific commits on a new branch and then force pushing would work.

**Comment by Influence365** [comment]
there we go, hopefully that worked

File: src/components/itemContextMenu.js [hunk 1]
```diff
@@ -381,28 +391,28 @@ define(['apphost', 'globalize', 'connectionManager', 'itemHelper', 'appRouter',
...
+                    import('imageEditor').then(({default: imageEditor}) => {
                         imageEditor.show({
                             itemId: itemId,
                             serverId: serverId
                         }).then(getResolveFunction(resolve, id, true), getResolveFunction(resolve, id));
                     });
                     break;
                 case 'identify':
-                    require(['itemIdentifier'], function (itemIdentifier) {
+                    import('itemIdentifier').then(({default:itemIdentifier }) => {
```
Comment (dkanada) [inline src/components/itemContextMenu.js:410]:
```suggestion
                    import('itemIdentifier').then(({default: itemIdentifier}) => {
```

**Pull request merged**